### DISCUSSION TITLE
Any plan to support redirect network packet cross network namespace

### DISCUSSION THREAD

**Discussion by neoareslinux** [cilium/ebpf#1322] [Q&A] [answered]
HI everyone:

i have a project, which need to redirect network packet from one interface(PHYSICAL) to another interface(TAP), and the destination interface is in an isolated network namespace, as far as i know , ebpf does' not support this redirection path. function like bpf_redirect、 bpf_redirect_map、bpf_redirect_peer、bpf_redirect_neighbor all cannot do this, is there any plan to implement this?

**Comment by lmb** [comment, marked as answer]
Hi, this isn't something that the library can help you with, really. Try emailing bpf@vger.kernel.org.
